### PR TITLE
Add sandstone stairs

### PR DESCRIPTION
**Pull request by provsalt**

### CODE REVIEW

File: server/block/sandstone_stairs.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+)
+
+// SandstoneStairs are blocks that allow entities to walk up blocks without jumping. They are crafted using end bricks.
```
Comment (Sandertv) [inline server/block/sandstone_stairs.go:11]: Crafted using end bricks?

**Review by Sandertv — CHANGES_REQUESTED** [review]
Just a minor documentation flaw, looks good otherwise.

File: server/block/sandstone_stairs.go (hunk 1 above)
Comment (provsalt) [reply server/block/sandstone_stairs.go]: woops haha

**Review by Sandertv — APPROVED** [review]
Looks good, thanks for your PR!

**Pull request merged**